src/routes/learn/seeding-nmf.svx

```diff
@@ -7,7 +7,175 @@ tags:
     - non-negative matrix factorization
     - basesMode
     - activations
+    - actMode
+    - nmf
+    - bufnmf
+    - seed
 flair: article
 ---
 
-## Seeding BufNMF with Bases and/or Activations
+<script>
+    import IndentNote from '$lib/components/IndentNote.svelte';
+    import Image from '$lib/components/Image.svelte';
+    import CodeBlock from '$lib/components/CodeBlock.svelte';
+    import { Tabs, TabList, TabPanel, Tab } from '$lib/components/tabs/tabs';
+    import Audio from '$lib/components/Audio.svelte';
+    // import ResourceLink from '$lib/components/ResourceLink.svelte';
+</script>
+
+<IndentNote type="pointer">
+
+This overview builds on information found in the [BufNMF Overview](/learn/bufnmf). Please visit that page first to learn more.
+
+</IndentNote>
+
+Non-negative matrix factorisation can be "seeded," meaning that it is given some information to begin the process from. You can provide bases and/or activations that you think will "point the algorithm" in the direction toward what you're hoping it will decompose. Without seeding, NMF activations and bases are both initialised with random values.
```

Comment:
I would remind them of the process, even a key word like "decomposition" might be helpful here